Assignment 1/Exercise2.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Exercise2
+ */
+import java.io.Console;
+
+public class Exercise2 {
+
+    public static void main(String[] args) {
+        Console console = System.console();
+        int year = Integer.parseInt(console.readLine("Type a year: "));
+        int century = year / 100;
+        if (year % 100 != 0) 
+          century = century +1;
```

Comment:
Awesome work!, in general the only thing I would like to suggest is to run code formater so all is correctly aligned, in vs code you can press `shift + alt + f` on each file.

This will be required for the next task but is optional for this one :)